packages/on-chain/README.md

```diff
@@ -1,57 +1,119 @@
 # Impact Evaluator Contracts
 
+## Description
+
+This project includes all contracts necessary for conducting an airdrop through Zodiac Reality, as well as tasks to facilitate contract interactions. The package can be used to deploy and configure a Zodiac reality module (found in `/tasks/reality`) and a Shrine contract (found in `tasks/shrine`).
```

Comment:
Is this a duplicate sentence? I'm on my phone so I might be misinterpreting.

Reply:
Changed